testing/engine/setvar.go

```diff
@@ -37,6 +37,6 @@ SecRule ARGS_NAMES "@rx ." \
 	nolog,\
 	setvar:'TX.paramcounter_%{MATCHED_VAR_NAME}=+1'"
```

Comment:
I think this test was succeeding because paramcounter_args_names coincidentally result to be the counter of unique parameter names, not the repetitions of each name.